test/e2e/es/mutation_test.go

```diff
@@ -79,7 +79,7 @@ func TestMutationResizeMemoryUp(t *testing.T) {
 			},
 		})
 
-	test.RunMutation(t, b, mutated)
+	test.RunMutation(t, b, mutated, test.MutationOptions{IncludesRollingUpgrade: true})
```

Comment:
IIUC `IncludesRollingUpgrade: true` creates an index with 1 replica. Is there only 1 data node in this test ? I think cluster health will be yellow.

Reply:
Fixing this in a follow-up PR.